### ISSUE TITLE
Sprockets 3.7.0 has deprecated methed `register_engine`

### ISSUE THREAD

**Issue by bfcoder** [emberjs/ember-rails#529]
Getting this warning with sprockets 3.7.0

```
DEPRECATION WARNING: Sprockets method `register_engine` is deprecated.
Please register a mime type using `register_mime_type` then
use `register_compressor` or `register_transformer`.
https://github.com/rails/sprockets/blob/master/guides/extending_sprockets.md#supporting-all-versions-of-sprockets-in-processors
 (called from setup at ember-es6_template-0.4.4/lib/ember/es6_template/sprockets3.rb:10)
DEPRECATION WARNING: Sprockets method `register_engine` is deprecated.
Please register a mime type using `register_mime_type` then
use `register_compressor` or `register_transformer`.
https://github.com/rails/sprockets/blob/master/guides/extending_sprockets.md#supporting-all-versions-of-sprockets-in-processors
 (called from setup at ember-es6_template-0.4.4/lib/ember/es6_template/sprockets3.rb:11)
DEPRECATION WARNING: Sprockets method `register_engine` is deprecated.
Please register a mime type using `register_mime_type` then
use `register_compressor` or `register_transformer`.
https://github.com/rails/sprockets/blob/master/guides/extending_sprockets.md#supporting-all-versions-of-sprockets-in-processors
 (called from setup at ember-es6_template-0.4.4/lib/ember/es6_template/sprockets3.rb:12)
DEPRECATION WARNING: Sprockets method `register_engine` is deprecated.
Please register a mime type using `register_mime_type` then
use `register_compressor` or `register_transformer`.
https://github.com/rails/sprockets/blob/master/guides/extending_sprockets.md#supporting-all-versions-of-sprockets-in-processors
 (called from setup at ember-es6_template-0.4.4/lib/ember/es6_template/sprockets3.rb:13)
```

Looks like ember-es6_template 0.5.3 has fixed this. Any chance you can upgrade from 0.4.4 to 0.5.3?

**Comment by bfcoder** [comment]
Looks like you are trying in https://github.com/emberjs/ember-rails/pull/511. Any headway on that?

**Comment by klippx** [comment]
+1

**Comment by howdoicomputer** [comment]
I forked the codebase and unpinned ember-es6_template (which grabbed 0.5.3) and I wasn't able to get it working with Rails 5 so... eh.

**Comment by efigarolam** [comment]
Did the same than @howdoicomputer for Rails 4.2.7  and it does not works neither. Got a `uninitialized constant Sprockets::Engines (NameError)` when starting server.

**Comment by tricknotes** [comment]
Sorry for my delay.
Now I published a new version of [ember-rails@0.21.0](https://rubygems.org/gems/ember-rails/versions/0.21.0).
The new version will avoid this warning to use ember-es6_template >= 0.5.
